tools/dev_console/static/index.html

```
@@ -315,23 +317,23 @@ <h2>Dynamic Pages</h2>
           </div>
 
           <div id="dynamic-summary" class="health-summary"></div>
-          <section class="dynamic-workspace">
+          <section class="tool-body dynamic-workspace">
             <aside>
               <input id="dynamic-filter" type="search" placeholder="Filter dynamic pages">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)